wbst-thry/src/app/graphics/lnr-graphics/playground.ts

```diff
@@ -22,23 +36,36 @@ export class Playground {
         skyBoxMaterial.reflectionTexture = new Babylon.CubeTexture('../../assets/blue/blue', scene);
         skyBoxMaterial.reflectionTexture.coordinatesMode = Babylon.Texture.SKYBOX_MODE;
         skyBox.renderingGroupId = 0;
+    }
 
-        // set up central planet
+    private static buildCentralPlanet(scene: Babylon.Scene, highlightLayer: Babylon.HighlightLayer): void {
         let planet = Babylon.MeshBuilder.CreateSphere('planet', { segments: 16, diameter: 4 }, scene);
         planet.renderingGroupId = 1;
+        let planetMaterial = new Babylon.StandardMaterial('planetMaterial', scene);
+        planetMaterial.diffuseTexture = new Babylon.Texture('../../assets/textures/planet1.jpg', scene);
```

Comment:
Haven't picked a texture yet, if we want it based on Nasa images we need to do some photo editing. We could also generate particles and make the "sun" ourselves. [Babylon has support for that](https://www.babylonjs-playground.com/#1VGT5D#2).